### ISSUE TITLE
ReadMe

### ISSUE THREAD

**Issue by shaoxuan92** [amueller/segmentation#1]
Hello, amueller, would you please write a brief introduction of running the code and the objective of it if it is convenient to you? Thank you!

**Comment by amueller** [comment]
yeah I'd love to do that some time, but honestly it's not a high priority for me right now.

**Comment by shaoxuan92** [comment]
Thank you very much for your reply. I'm wondering if you could help me in obstacles now I am encountering. I just wanna get the BoW of each segmentation. How should I do？ Or Is there any algorithm that is based on BoW, which I mean in the end, I could get the segmentation results and BoW of each segmentation. Thank you very much if you would take a while for help~_~

**Comment by shaoxuan92** [comment]
And by the way, You are a Data Science major in NYU? One of my classmates called Ren Zizhuo  is studying the same as you

**Comment by amueller** [comment]
I'm faculty ;) I did my phd in 2013.
